### PR TITLE
Use release timestamp instead of build date

### LINKED ISSUE #1220: Releases show time always as midnight while appearing to have higher resolution than "day"
Plugin site right now:
> Screenshot 2022-08-11 at 21 41 33

Jenkins right now:
> Screenshot 2022-08-11 at 21 41 39

I was genuinely wondering whether that's the same release until I noticed that the tooltip on the plugin site says the release was at 00:00.

Unsure what the data source is, but if it only has day resolution, the human-friendly labels should just say "today", "yesterday", "X days ago" instead of "X hours ago", and the tooltip shouldn't give the time.

### PR DESCRIPTION
**Pull request by zbynek**
Fixes #1220

### CODE REVIEW

**Comment by zbynek** [comment]
CC @daniel-beck

File: plugins/gatsby-source-jenkinsplugins/utils.js [hunk 1]
```diff
@@ -472,13 +472,14 @@ const fetchPluginVersions = async ({createNode, reporter, firstReleases}) => {
             url    "https://updates.jenkins.io/download/plugins/42crunch-security-audit/2.1/42crunch-security-audit.hpi"
             version 2.1
             */
-            const date = parseDate(data.buildDate, 'MMMM d, yyyy', new Date(0));
```
Comment (halkeye) [inline plugins/gatsby-source-jenkinsplugins/utils.js:475]: yay! only one last date parsing left.

**Comment by halkeye** [comment]
seems good to me. Keep an eye on it after next deploy? Do you have/want sentry access (there's a lot of noise)?

**Comment by zbynek** [comment]
I have Sentry access.

**Pull request merged**